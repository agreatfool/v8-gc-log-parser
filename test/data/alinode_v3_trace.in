[77580:0x102801600]     2195 ms: Scavenge 12.0 (18.9) -> 8.2 (18.9) MB, 0.4 / 0.0 ms  allocation failure 
[77580:0x102801600]     2201 ms: Mark-sweep 8.2 (18.9) -> 5.9 (18.9) MB, 0.6 / 0.0 ms  (+ 5.1 ms in 10 steps since start of marking, biggest step 1.1 ms, walltime since start of marking 6 ms) finalize incremental marking via task GC in old space requested
[77580:0x102801600]     2322 ms: Scavenge 10.0 (18.9) -> 6.1 (18.9) MB, 0.2 / 0.0 ms  allocation failure 
