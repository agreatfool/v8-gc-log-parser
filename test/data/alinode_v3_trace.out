[
  {
    "pid": 77580,
    "isolate": "0x102801600",
    "time_since_init": 2195,
    "data": {
      "collector": "scavenge",
      "start_object_size": 12,
      "end_object_size": 8.2,
      "start_memory_size": 18.9,
      "end_memory_size": 18.9,
      "pause": 0.4,
      "external_time": 0
    },
    "type": "trace"
  },
  {
    "pid": 77580,
    "isolate": "0x102801600",
    "time_since_init": 2201,
    "data": {
      "collector": "mark_sweep",
      "start_object_size": 8.2,
      "end_object_size": 5.9,
      "start_memory_size": 18.9,
      "end_memory_size": 18.9,
      "pause": 0.6,
      "external_time": 0,
      "step_description": "+ 5.1 ms in 10 steps since start of marking, biggest step 1.1 ms, walltime since start of marking 6 ms"
    },
    "type": "trace"
  },
  {
    "pid": 77580,
    "isolate": "0x102801600",
    "time_since_init": 2322,
    "data": {
      "collector": "scavenge",
      "start_object_size": 10,
      "end_object_size": 6.1,
      "start_memory_size": 18.9,
      "end_memory_size": 18.9,
      "pause": 0.2,
      "external_time": 0
    },
    "type": "trace"
  }
]
